date_validator.gemspec

```diff
@@ -5,20 +5,19 @@ require "date_validator/version"
 Gem::Specification.new do |s|
   s.name        = "date_validator"
   s.version     = DateValidator::VERSION
-  s.platform    = Gem::Platform::RUBY
   s.authors     = ["Oriol Gual", "Josep M. Bach", "Josep Jaume Rey"]
   s.email       = ["info@codegram.com"]
   s.homepage    = "http://github.com/codegram/date_validator"
-  s.summary     = %q{A simple, ORM agnostic, Ruby 1.9 compatible date validator for Rails 3, based on ActiveModel.}
-  s.description = %q{A simple, ORM agnostic, Ruby 1.9 compatible date validator for Rails 3, based on ActiveModel. Currently supporting :after, :before, :after_or_equal_to and :before_or_equal_to options.}
+  s.summary     = %q{A simple, ORM agnostic, Ruby 1.9 compatible date validator for Rails 3+, based on ActiveModel.}
+  s.description = %q{A simple, ORM agnostic, Ruby 1.9 compatible date validator for Rails 3+, based on ActiveModel. Currently supporting :after, :before, :after_or_equal_to and :before_or_equal_to options.}
 
   s.rubyforge_project = "date_validator"
 
-  s.add_runtime_dependency 'activemodel'
-  s.add_runtime_dependency 'activesupport'
+  s.add_runtime_dependency 'activemodel', '>= 3'
+  s.add_runtime_dependency 'activesupport', '>= 3'
 
   s.add_development_dependency 'minitest'
-  s.add_development_dependency 'rake'
+  s.add_development_dependency 'rake', '12.2.1'
```

Comment:
Maybe we could drop support for Ruby 1.9.3 on newer versions? /cc @oriolgual

Reply:
It's no big deal to have tightly specified development dependencies, so if you guys are happy with this (otherwise very uncontroversial) PR I'd rather keep it separate from the thornier discussion of whether you want to support Ruby 1.9.3...

Reply:
Sure, no problem! Still, I'd like to wait for someone else's review before merging this if you don't mind 😄

Reply:
Oh, and thanks for your PR! 😁